pkg/repositories/config/migrations.go

```diff
@@ -445,6 +445,26 @@ var LegacyMigrations = []*gormigrate.Migration{
 			return tx.Model(&models.Execution{}).Migrator().DropColumn(&models.Execution{}, "launch_entity")
 		},
 	},
+	// Create admin tags table.
+	{
+		ID: "2023-06-01-admin_tags",
+		Migrate: func(tx *gorm.DB) error {
+			return tx.AutoMigrate(&models.AdminTag{})
```

Comment:
nope not this.  never again hahaha

Reply:
yeah, let's not do this. The problem with using model files directly is that we can't track changes done to the underlying model (think of the case users are updating from different versions of flyteadmin). 

The template we've been using in this file is to declare the model to be migrated inline, e.g. https://github.com/flyteorg/flyteadmin/blob/master/pkg/repositories/config/migrations.go#L897-L920. Notice how the `ExecutionEvent` model is defined in the same scope as the migration.